addons/web_unsplash/controllers/main.py

```diff
@@ -81,7 +81,7 @@ def slugify(s):
         for key, value in unsplashurls.items():
             url = value.get('url')
             try:
-                if not url.startswith('https://images.unsplash.com/'):
+                if not url.startswith('https://images.unsplash.com/') and not url.startswith('https://plus.unsplash.com/'):
```

Comment:
Maybe this is more concise ?
```suggestion
                if not url.startswith(('https://images.unsplash.com/', 'https://plus.unsplash.com/')):
```